### PR TITLE
Add retryAnalyzer

### PR DESCRIPTION
**Pull request by GiannaGrale**

### CODE REVIEW

File: src/main/java/retryAnalyzer/Retry.java [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+import org.testng.IRetryAnalyzer;
+import org.testng.ITestResult;
+
+/***
+ * RetryAnalyzer helps to check if the failure was caused by a product bug or another reason.
+ */
+public class Retry implements IRetryAnalyzer {
+    private int attempt = 0;
+    private static final int MAX_RETRY = 4;
```
Comment (Anassstasi) [inline src/main/java/retryAnalyzer/Retry.java]: Please update this value to 2, 4 is to much

File: src/main/java/retryAnalyzer/Retry.java [hunk 2]
```diff
@@ -0,0 +1,28 @@
+package retryAnalyzer;
+
+import org.testng.IRetryAnalyzer;
+import org.testng.ITestResult;
+
+/***
+ * RetryAnalyzer helps to check if the failure was caused by a product bug or another reason.
```
Comment (Anassstasi) [inline src/main/java/retryAnalyzer/Retry.java]: In case of a bug test should not be rerun

File: src/main/java/retryAnalyzer/Retry.java (hunk 1 above)
Comment (GiannaGrale) [reply src/main/java/retryAnalyzer/Retry.java]: done

File: src/main/java/retryAnalyzer/Retry.java (hunk 2 above)
Comment (GiannaGrale) [reply src/main/java/retryAnalyzer/Retry.java]: the java doc is changed

**Review by Anassstasi — APPROVED** [review]
LGTM!

**Pull request merged**